### DISCUSSION TITLE
Individual Licence

### DISCUSSION THREAD

**Discussion by clt-pereira** [datalust/seq-tickets#2335] [Help]
Good night everyone!

I have doubts about what would be “server installer” mentioned in the Team licensing and not mentioned in the Individual license, for example.
As for installation on a personal server and not on the client's server, this type of blocking is conditioned in some way during the installation using the individual license, that is, I provide software in the SaaS model and it is hosted on my own server. enterprise on Windows, can I provide this software to customers and still use the individual license to use Seq?

**Comment by nblumhardt** [comment]
Hi @clt-pereira; I'll follow up on your other ticket https://github.com/datalust/seq-tickets/discussions/2332 to keep the discussion thread intact. Cheers!
